oap-server/server-starter/src/main/java/org/apache/skywalking/oap/server/starter/config/ApplicationConfigLoader.java

```diff
@@ -40,6 +43,15 @@
  */
 @Slf4j
 public class ApplicationConfigLoader implements ConfigLoader<ApplicationConfiguration> {
+    static final Set<String> SENSITIVE_PROPERTY_NAMES =
+        ImmutableSet.<String>builder()
+                    .add("password")
+                    .add("authentication")
+                    .add("accessKey")
+                    .add("secretKey")
```

Comment:
NIT: Check against https://skywalking.apache.org/docs/main/next/en/setup/backend/configuration-vocabulary/ and I think 1 is missing.
```suggestion
                    .add("secretKey")
                    .add("firehoseAccessKey")
```

Reply:
@toffentoffen thank you!!